Nodejs/Product/Nodejs/Project/NodejsProjectLauncher.cs

```diff
@@ -315,13 +339,45 @@ private void StartWithDebugger(string startupFile)
         {
             var dbgInfo = new VsDebugTargetInfo();
             dbgInfo.cbSize = (uint)Marshal.SizeOf(dbgInfo);
-
             if (SetupDebugInfo(ref dbgInfo, startupFile))
             {
                 LaunchDebugger(this._project.Site, dbgInfo);
             }
         }
 
+        private void StartWithChromeV2Debugger(string program)
+        {
+            string cwd = Path.GetDirectoryName(program);
+            JObject configuration = new JObject(
+                new JProperty("name", "Debug Node.js program from Visual Studio"),
```

Comment:
I sent an e-mail to Rob about this, I'll send a new PR if we need to change that.